File: strings/rafaelpetrucci/1551.py
alfabeto = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
total = len(alfabeto)
number = int(input())
for _ in range(number):
    frase = input()
    aux = alfabeto.copy()
    for alfa in alfabeto:
        if alfa in frase:
            aux.remove(alfa)
    valor_Aux = len(aux)
    if valor_Aux == 0:
        print('frase completa')
    elif valor_Aux <= total//2:
        print("frase quase completa")
    else:
        print("frase mal elaborada")

File: strings/lucianoclaudio/1551.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    string a;
    int n;

    cin >> n;
    cin.ignore(INT_MAX, '\n');
    for (int i=0;i<n;i++){
        int l=0;
        getline(cin,a);
        string alfa="abcdefghijklmnopqrstuvwxyz";
        for (int u=0;u<a.size();u++){
            for (int j=alfa.size();j>=0;j--){
                if (a[u]==alfa[j]){
                    l++;
                    alfa.erase(alfa.begin()+j);
                    if (j==0){
                        break;
                    }
                }
            }
        }
        if (l==26){
            cout << "frase completa\n";
        }
        else{
            if (l>13){
                cout << "frase quase completa\n";
            }
            else{
                cout << "frase mal elaborada\n";
            }
        }
    }
    return 0;
}